core/color/color_test.go

```diff
@@ -41,3 +41,37 @@ func TestOpposite(t *testing.T) {
 		})
 	}
 }
+
+func TestFromSGFProp(t *testing.T) {
+	out, err := FromSGFProp("B")
+	if err != nil {
+		t.Fatal(err)
+	}
+	if out != Black {
+		t.Errorf("got %v, but wanted %v", out, Black)
+	}
+
+	out, err = FromSGFProp("AB")
+	if err != nil {
+		t.Fatal(err)
+	}
+	if out != Black {
+		t.Errorf("got %v, but wanted %v", out, Black)
+	}
+
+	out, err = FromSGFProp("W")
+	if err != nil {
+		t.Fatal(err)
+	}
+	if out != Black {
+		t.Errorf("got %v, but wanted %v", out, White)
+	}
+
+	out, err = FromSGFProp("AW")
```

Comment:
Can you add a test for an error-case? In other words, passing in something like "Z" or ""?

Reply:
Will do!